static/app/utils/analytics/discoverAnalyticsEvents.tsx

```diff
@@ -1,11 +1,29 @@
 export type DiscoverEventParameters = {
+  'discover_v2.add_equation': {};
+  'discover_v2.build_new_query': {};
+  'discover_v2.change_sort': {sort: string};
+  'discover_v2.facet_map.clicked': {tag: string};
+  'discover_v2.prebuilt_query_click': {query_name: string | undefined};
```

Comment:
@roggenkemper You can do `{query_name?: string}` if i'ts optional

Reply:
fixed!